tools/pr-docs

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash -e
+# Distributed under the MIT License.
+# See LICENSE.txt for details.
+
+if [ "$#" -ne 2 ] ; then
+    echo "Usage: $0 pr-number output/location" >&2
+    echo "Render documentation from a pull request" >&2
```

Comment:
I think this needs more documentation. Where can I run it from (I'd just say people should run it from SPECTRE_HOME to make your life easier)? Will it set up a full build tree or just dump the docs right into the output directory?

I'm not sure where in the Doxygen documentation we should point people to this file. Maybe the code review guidelines?